### DISCUSSION TITLE
chia plots check

### DISCUSSION THREAD

**Discussion by uldh** [Chia-Network/chia-blockchain#6955] [Support]
Does anyone know why "chia plots check" report those "ERROR" ? i have 1000 plots ERROR reported, but those 1000 plots works well now.
https://user-images.githubusercontent.com/60197354/120218073-371ae080-c207-11eb-9ec3-3690dc2fbe1a.png

**Comment by uldh** [comment]
I found the answer, those "ERROR" is caused by third-party pool, close the pool, and errors will disappear. 
I hope this can help others.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
